### ISSUE TITLE
Support Descriptive Video Service (Descriptive Audio Track)

### ISSUE THREAD

**Issue by flavioribeiro** [kaltura/nginx-vod-module#1049]
According to the HLS specification, we must sinalize that a given audio track is DVS by the addition of `CHARACTERISTICS="public.accessibility.describes-video"` tag. I wonder if we should implement an option at the Sequence level on mapped mode where I set `"dvs": true` or something and that propagates all the way to the HLS manifest. Thoughts on this one @erankor?

More information about this one: https://developer.apple.com/documentation/http_live_streaming/hls_authoring_specification_for_apple_devices

**Comment by erankor** [comment]
Maybe it makes more sense to generalize it a bit, and instead of boolean dvs, have an enum of `role` (I believe that is the DASH parallel of this). If we use `role`, the same field could also be used to differentiate between SUBTITLES and CLOSED-CAPTIONS (there's some other issue on that topic...)

**Comment by flavioribeiro** [comment]
Yes, that makes sense. Linking https://github.com/kaltura/nginx-vod-module/issues/825 and https://github.com/kaltura/nginx-vod-module/issues/996 to this.

**Comment by Olivier6767** [comment]
Any idea when the issue with the CC button on iOS player will be fixed?
[https://developer.apple.com/library/archive/qa/qa1801/_index.html](url)

Thx

**Comment by dio-az** [comment]
This is a quite old issue, is it part of any roadmap?
